/**:
  ros__parameters:

    sweeping_destination : [-31.05178451538086, -26.928119659423828, 0.0,
                            0.0, 0.0, 0.8112418670095116, 0.5847107260953247]

    supply_station: [213.5743408203125, 102.43115997314453, 0.0, 
                     0.0, 0.0, 0.6387085974241705, 0.7694487166643719]

    garbage_dump: [218.26470947265625, 101.36994171142578, 0.0, 
                   0.0, 0.0, 0.6332124229087648, 0.7739780536126406]

    preset_points: [5.0, 0.0, 0.0,
                    5.0, 5.0, 0.0,
                    5.0, -5.0, 0.0]
